pkg/command/helpers_test.go

```diff
@@ -66,34 +62,10 @@ func testSleepBackoff(t *testing.T, i int) {
 func testCloneRepository(t *testing.T, url, username, password, path, branchName string) {
 	t.Helper()
 
-	auth := testBasicAuthMethod(t, username, password)
-
-	_, err := git2go.Clone(url, path, &git2go.CloneOptions{
-		FetchOptions: &git2go.FetchOptions{
-			DownloadTags: git2go.DownloadTagsNone,
-			RemoteCallbacks: git2go.RemoteCallbacks{
-				CredentialsCallback: auth.CredCallback,
-			},
-		},
-		CheckoutBranch: branchName,
-	})
+	err := git.Clone(url, username, password, path, branchName)
```

Comment:
As noted before I'm wondering about why this have been moved out to the application code

Reply:
It's being used by tests in both the command and git packages.